crd/multitenancy/api/v1alpha1/utils.go

```diff
@@ -0,0 +1,30 @@
+package v1alpha1
+
+// IsReady checks if all the required fields in the MTPNC status are populated
+func (m *MultitenantPodNetworkConfig) IsReady() bool {
+	if m.Status.PrimaryIP == "" ||
+		m.Status.MacAddress == "" ||
+		m.Status.NCID == "" ||
+		m.Status.GatewayIP == "" {
+		return false
+	}
```

Comment:
Right now, these are deprecated, and we move everything into the `InterfaceInfos`. `InterfaceInfos` is array and MTPNC is a single value.
Do we still need to check if all these fields in the MTPNC status are populated?

Reply:
yes, we need to double check it;